lib/rules/at-rule-descriptor-no-unknown/__tests__/index.mjs

```diff
@@ -0,0 +1,80 @@
+import rule from '../index.mjs';
+const { messages, ruleName } = rule;
+
+testRule({
+	ruleName,
+	config: true,
+
+	accept: [
+		{
+			code: 'a { unknown: 0; }',
+			description: 'ignore normal properties',
+		},
+		{
+			code: '@counter-style foo { system: cyclic; symbols: url(bar.svg); suffix: " "; }',
+		},
+		{
+			code: '@property --foo { syntax: "<integer>"; inherits: false; initial-value: 0; }',
+		},
+		// TODO: The following test can be supported once mdn/data, is updated and the css-tree is updated.
+		// see: https://github.com/mdn/data/blob/6b4cae7c0aa96c199a982f2b6efeda7ce67eb515/css/at-rules.json
+		{
+			code: '@view-transition { navigation: auto; }',
+			skip: true,
+		},
```

Comment:
```suggestion
		{
			code: '@view-transition { navigation: auto; }',
		},
		{
			code: '@foo { bar: 0 }',
		},
```

We should ignore these as an unknown (to CSSTree).

Reply:
Ah, I see, that does seem more natural. Thank you for the suggestion!

Reply:
Fixed in this commit.
[Skip unknown at-rules](https://github.com/stylelint/stylelint/pull/8197/commits/d44e66cbe99f7c57e663d196e4642c9cdc264ab3)